assets/src/components/editing/elements/video/VideoCommandEditor.tsx

```diff
@@ -1,43 +1,54 @@
 import React, { useEffect, useMemo } from 'react';
 import { MessageEditorComponent } from '../command_button/commandButtonTypes';
 
-const startEndCueRegex = /startcuepoint=([0-9]+).([0-9]+);endcuepoint=([0-9]+).([0-9]+)/;
-const startCueRegex = /startcuepoint=([0-9]+).([0-9]+)/;
+const startEndCueRegex = /startcuepoint=([0-9.]+);endcuepoint=([0-9.]+)/;
+const startCueRegex = /startcuepoint=([0-9.]+)/;
 
 // This is the editor for the command that gets sent from the command button to the video player.
 export const VideoCommandEditor: MessageEditorComponent = ({ onChange, value }) => {
-  const [startSec, setStartSec] = React.useState(0);
-  const [startSubSec, setStartSubSec] = React.useState(0);
-  const [endSec, setEndSec] = React.useState(0);
-  const [endSubSec, setEndSubSec] = React.useState(0);
+  const [startSec, setStartSec] = React.useState('');
```

Comment:
Switched from handling these as numbers to strings to better handle the user clearing the input field and putting in an empty string to signify "no cue point"